setup.py

```diff
@@ -194,7 +194,7 @@ def extract_version():
 
 
 setup(
-    name='Iris',
+    name='iris-scitools',
```

Comment:
I think I prefer `iris_scitools` with 'iris' at the start. If the packages were sorted alphabetically, and someone looked for `iris` they'd find `iris_scitools` nearby.